docs/UserGuide.md

```diff
@@ -125,189 +156,285 @@ An invocation of the command is exemplified by:
+Lists all available mappings, paginated if dataset is too huge.
+**Format:** `mapping list`
 
+**Example:**
 | `mapping list`                               |
 |----------------------------------------------|
 | ![mapping-list-ui](images/MappingListUi.png) |
 
-### Add a mapping: `mapping add`
-
-Adds mapping for local course identified by `localcode`, offered by partner `university`, as course with code 
+### 4.4.2. Add a mapping: `mapping add`
+Adds mapping for local course identified by `localcode`, offered by partner `university`, as course with code
 `partnercode`. You will be prompted to create courses which codes you specify, but do not exist in the database.
+**Format:** `mapping add  [localcode] [university] [partnercode]`
 
-Format: `mapping add [localcode] [university] [partnercode]`
-
-Example:
-
+**Example**:
 | `mapping add [CS3230] [University of Toronto] [CSC373H1]` |
 |-----------------------------------------------------------|
 | ![mapping-add-ui](images/MappingAddUi.png)                |
 
-### Delete a mapping : `mapping delete`
-
-Deletes mapping for local course identified by `localcode`, offered by partner `university`, as course with code 
-`partnercode`. 
-
-Format: `mapping delete [localcode] [university] [partnercode]`
-
-Example:
+### 4.4.3. Delete a mapping: `mapping delete`
+Deletes mapping for local course identified by `localcode`, offered by partner `university`, as course with code
+`partnercode`.
+**Format:** `mapping delete [localcode] [university] [partnercode]`
 
+**Example:**
 | `mapping delete [CS3230] [University of Toronto] [CSC373H1]` |
 |--------------------------------------------------------------|
 | ![mapping-delete-ui](images/MappingDeleteUi.png)             |
 
-### List all notes: `note list`
+### 4.4.4 Search a mapping by attributes: `mapping search`
+**Format:** `mapping search [localcode]` \
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; `mapping search [localname]` \
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; `mapping search [partnercode]` \
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; `mapping search [partnername]` \
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; `mapping search [university]` 
 
-Lists all notes, paginated if dataset is too huge.
+**Example:**:
 
-Format: `note list`
+### 4.4.5. Sort a mapping by attributes: `mapping sort`
+**Format:**  `mapping sort [localcode]` \
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; `mapping sort [localname]` \
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; `mapping sort [partnercode]` \
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; `mapping sort [partnername]` \
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; `mapping sort [university]` 
```

Comment:
Disclaimer: This is a temporary/quick fix for indentation. Will update if there's a better solution.